QEfficient/utils/generate_inputs.py

```diff
@@ -147,9 +163,13 @@ def update_ort_inputs(self, iteration, inputs, ort_outputs, n_layer):
     def prepare_cloud_ai_100_inputs(self, n_layer, padding_shape):
         """
         Function responsible for creating Prefill stage numpy inputs for ONNX model to be run on Cloud AI 100.
-        :param n_layer : int
-        :param padding_shape : List[int]
-        :return inputs: Dict - input_ids, position_ids, past_key_values
+        ---------
+        
+        :n_layer : int. Number of layers in the PyTorch model.
+        :padding_shape : List[int]. Shape of past key values.
+
+        Return:
+            inputs: Dict - input_ids, position_ids,attention_mask, past_key_values, cache_index

```

Comment:
Remove cache_index and attention_mask from all inputs, new interface with range based gather does not have these inputs.